polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -613,21 +617,27 @@ class CodeCell extends Cell {
             }
         }
         cellState.state.presence.forEach(p => updatePresence(p.id, p.name, p.color, p.range))
-        cellState.view("presence").addObserver((newPresence, oldPresence) => {
-            const removed = diffArray(newPresence, oldPresence)[1]
+        cellState.observeKey("presence", (newPresence, update) => {
+            const removed = Object.values(update.removedValues ?? {})
 
             removed.forEach(p => {
-                const marker = presenceMarkers[p.id]
-                if (marker) {
-                    this.editor.deltaDecorations(marker, [])
+                if (p) {
```

Comment:
What would cause `p` to be undefined?

Reply:
Reality

Reply:
lol, ok. Just not seeing what might be adding an `undefined` value into this array...